cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/strategy/PostponeUrpReplicaMovementStrategy.java

```diff
@@ -28,12 +32,14 @@ public Comparator<ExecutionTask> taskComparator(StrategyOptions strategyOptions)
             task1.proposal().topicPartition());
         task1PartitionExists = true;
       } catch (PartitionNotExistsException e) {
+        LOG.warn("Task 1 - Partition {} does not exist in cluster.", task1.proposal().topicPartition());
```

Comment:
The log can be further improved by changing to something like `LOG.warn("Task {} skipped comparison since partition {} does not exist in cluster.", task1, task1.proposal().topicPartition());`

Reply:
this would be much better. making the change